Formula/mtm.rb

```diff
@@ -0,0 +1,40 @@
+class Mtm < Formula
+  desc "Micro terminal multiplexer"
+  homepage "https://github.com/deadpixi/mtm"
+  url "https://github.com/deadpixi/mtm/archive/1.2.0.tar.gz"
+  sha256 "68f753f743fcd3d87f269011d4bbd3fed59cfcad710d7c7db98844e3e675c196"
+  license "GPL-3.0-or-later"
+
+  head do
+    url "https://github.com/deadpixi/mtm.git"
+
+    uses_from_macos "ncurses"
```

Comment:
Head can work with an older version of ncurses then the stable release?

Reply:
Oddly enough, that is exactly the case here. 🙃 
Upstream backported/got around some ncurses 6.1 stuff (`pair.c`) → https://github.com/deadpixi/mtm/commit/5944ba1